mono/tests/split-tailcall-interface-conservestack.cpp

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Remove this file.

Reply:
This file is critical to maintenance of the tests. I have used it repeatedly as I edit the tests and it saved me a ton of time.